storage/redis/peer_crud.go

```diff
@@ -0,0 +1,79 @@
+package redis
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/chihaya/chihaya/bittorrent"
+	redigo "github.com/garyburd/redigo/redis"
```

Comment:
We put third-party imports as a block below the standard library imports, and local imports at the end, as another block. We have a meta-package that illustrates this ([see here](https://github.com/jzelinskie/conventions/blob/master/conventions.go)).